### PR TITLE
net-p2p/syncthing: bump to 0.14.27

### PR DESCRIPTION
**Pull request by nickgen**
Minor release.

### CODE REVIEW

**Comment by sbraz** [comment]
@nickgen I'd like to stop maintaining this package, can you remove me from the metadata through this PR?

**Comment by mgorny** [comment]
@nickgen, do you want to take it over? If so, feel free to add yourself.

@djc, are you a co-maintainer here or proxy for @sbraz?

**Comment by djc** [comment]
@sbraz sad to see you go! Any particular reason for ending your maintenance work?

@nickgen I'll echo @mgorny, would be nice if you can take over.

@mgorny I was maintaining it before @sbraz was, but am just as happy not to. However, I will if no one else is able to.

**Comment by sbraz** [comment]
@djc I'm not using it any more and they bump it every other week, it's a bit tedious :/
I could keep maintaining it but I think I should try to focus on other packages, I know this one has a lot of users so it'll be easier to find a maintainer.

**Comment by nickgen** [comment]
I'm not skilled enough for being a maintainer right now, sorry. I've just started using Gentoo not so long ago.

@sbraz I'll remove you from metadata, if it's ok to do so, It's kinda confusing. For me it's like someone unknown came from nowhere, removed some maintainer from some package and disappeared.

**Comment by djc** [comment]
@sbraz makes total sense. @mgorny maybe it makes sense to send out a package up for grabs notice to gentoo-dev to ask who are interested?

@sbraz out of personal interest, have you replaced your usage of syncthing with something else, and if so, what?

**Comment by djc** [comment]
@nickgen I'm also happy to remove @sbraz from the metadata if you're more comfortable with that.

**Comment by nickgen** [comment]
@djc I'm comfortable to do so, if everyone else thinks, that's ok. I just wanted to make sure that this is normal.

@sbraz Done, removed you from metadata.

**Comment by sbraz** [comment]
@nickgen Thanks!
@djc The performance of the Android app just wasn't great :/

File: net-p2p/syncthing/metadata.xml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person">
-    <email>sautier.louis@gmail.com</email>
-    <name>Louis Sautier</name>
-  </maintainer>
   <maintainer type="project">
```
Comment (mgorny) [inline net-p2p/syncthing/metadata.xml]: Whoever merges this, please remember to also remove proxy-maint along with the last proxied maintainer.

**Comment by djc** [comment]
Rebased onto master, with 55ae50d2c80a86418bd4213228485c4ef28feec5 to remove the Proxy Maintainers project from metadata.

**Pull request closed without merging**